test/versioned/mysql/basic-pool.tap.js

```diff
@@ -251,21 +251,24 @@ tap.test('mysql built-in connection pools', function (t) {
   t.test('lack of callback does not explode', function (t) {
     helper.runInTransaction(agent, function transactionInScope(txn) {
       pool.query('SET SESSION auto_increment_increment=1')
-      txn.end()
-      t.end()
+      setTimeout(function () {
```

Comment:
I'd ported this in from the equivalent mysql2 test during debugging. It's not needed here, but it is needed by the mysql2 tests.  I'll delete it here, because the warning's misleading.